[2021-02-28 13:13:26,585] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [queued]>
[2021-02-28 13:13:26,589] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [queued]>
[2021-02-28 13:13:26,589] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:13:26,589] {taskinstance.py:1043} INFO - Starting attempt 2 of 2
[2021-02-28 13:13:26,589] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-28 13:13:26,594] {taskinstance.py:1063} INFO - Executing <Task(HttpSensor): is_api_available> on 2021-02-28T12:12:58.544973+00:00
[2021-02-28 13:13:26,597] {standard_task_runner.py:52} INFO - Started process 40214 to run task
[2021-02-28 13:13:26,606] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'bitcoin_price_dag', 'is_api_available', '2021-02-28T12:12:58.544973+00:00', '--job-id', '16', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/bitcoin_price_dag.py', '--cfg-path', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp42zlzmnb', '--error-file', '/var/folders/fh/z0rf64gj7d93005s7pxrkwm40000gn/T/tmp2gdfgb9u']
[2021-02-28 13:13:26,608] {standard_task_runner.py:77} INFO - Job 16: Subtask is_api_available
[2021-02-28 13:13:26,640] {logging_mixin.py:104} INFO - Running <TaskInstance: bitcoin_price_dag.is_api_available 2021-02-28T12:12:58.544973+00:00 [running]> on host 120.0.168.192.in-addr.arpa
[2021-02-28 13:13:26,661] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=bitcoin_price_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2021-02-28T12:12:58.544973+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-02-28T12:12:58.544973+00:00
[2021-02-28 13:13:26,662] {http.py:102} INFO - Poking: ticker
[2021-02-28 13:13:26,666] {base.py:65} INFO - Using connection to: id: bitcoin_api. Host: http://blockchain.info/, Port: None, Schema: , Login: , Password: None, extra: None
[2021-02-28 13:13:26,669] {http.py:140} INFO - Sending 'GET' to url: http://blockchain.info/ticker
[2021-02-28 13:13:26,968] {local_task_job.py:146} INFO - Task exited with return code Negsignal.SIGABRT
